server/clip_server/torch-flow.yml

```diff
@@ -8,6 +8,6 @@ executors:
       jtype: CLIPEncoder
       metas:
         py_modules:
-          - clip_server.executors.clip_torch
+          - clip_server.executors.clip_onnx
```

Comment:
Why this change? Should the onnx model be loaded with onnx-flow.yml?

Reply:
Sorry, it was for tests and added by mistake.